src/custom/components/analytics/events/transactionEvents.ts

```diff
@@ -68,6 +68,11 @@ export function signSwapAnalytics(orderClass: OrderClass, label?: string) {
 
 export type OrderType = 'Posted' | 'Executed' | 'Canceled' | 'Expired'
 export function orderAnalytics(action: OrderType, orderClass: OrderClass, label?: string) {
+  if (action === 'Posted') {
+    window.fbq?.('track', 'Lead')
+    window.lintrk?.('track', { conversion_id: 10759522 })
```

Comment:
Where are these ids coming from?

Reply:
They are defined in this file https://docs.google.com/spreadsheets/d/1GFg-gXmBboLGLeLsTGA_7FxmuCB3P0xNbdwHhEEe7d4/edit?pli=1#gid=1015340718, but I assume they are generated on the side of the Linkedin analytics console

Reply:
I think they should be moved to an util per social media, where we expose the sendEvent method, and the constants with nice name, so we don't use this un-intuitive names here, and we don't use the window object here either (this should be hidden and encapsulated in a lib/util with a link to the spreadsheet)